src/test/files/MakeNonReassignedVariablesConstants.java

```diff
@@ -1,9 +1,15 @@
 import java.util.logging.Logger;
+import lombok.Setter;
```

Comment:
this is the first use case ... using real imports like that.
but there is a second use case missing : if you don't use import but "@lombok.Setter" directly for example
could you add this second use case for all lombok imports ? you can add a second test file and a second unit test to check it.